src/main/java/org/springframework/data/redis/connection/jedis/JedisConnectionFactory.java

```diff
@@ -125,8 +126,7 @@ public class JedisConnectionFactory
 
 	private @Nullable RedisConfiguration configuration;
 
-	private RedisStandaloneConfiguration standaloneConfig = new RedisStandaloneConfiguration("localhost",
-			Protocol.DEFAULT_PORT);
+	private RedisStandaloneConfiguration standaloneConfig = defaultStandaloneConfiguration();
```

Comment:
Introducing a indirection makes it difficult to understand at first sight the details.

Reply:
Same reason as above.